dexbot/strategies/base.py

```diff
@@ -1034,6 +1034,43 @@ def is_sell_order(self, order):
         else:
             return False
 
+    def is_too_small_amounts(self, amount_quote, amount_base):
```

Comment:
These are generic methods intended to be reused from another strategies.

Reply:
May I suggest that we move methods into base.py once they are used in at least two strategies? Then it would make sense and also require no discussion.

Reply:
@thehapax @MarkoPaasila would you be happy if I'll refactor RO and SO to use these methods? RO and SO currently have the similar code inside (both methods). So there is x3 code doubling. Of course, I would refactor SO and RO in separate PR.

Moving these methods into KOTH, then back into base.py not seems useful task.

Reply:
@bitfag @MarkoPaasila we do need to refactor the strategies. I am not against having the base strategy handle order logic but we need to consider the base.py now has 52 methods (and increasing!!)  and needs to be refactored to become a manageable class. There are a lot of market query methods in the base.py and will discuss on slack with joelva about how we can separate some of the base.py methods out, not necessarily this one.